### DISCUSSION TITLE
JSLT + Apache NiFi

### DISCUSSION THREAD

**Discussion by contatolucascosta** [schibsted/jslt#249] [Ideas]
Hi.

What do you think create a JSLT processor to Apache NiFi ?
I'm already using JSLT in my own Json/Rest API and also Apache Camel component.

Recently I discovery and test the Apache NiFi, that has a lot of processors.
There is the JOLT transformations, but for sure I prefer JSLT language!

Thanks.
Lucas Costa

**Comment by larsga** [comment]
I think it's a great idea! I'd be very happy to see you do that. If you have question about how I'll be happy to answer those, too. (Although I don't know any Nifi details.)

**Reply by contatolucascosta** [reply]
All right!
I’ll study how to do that and I keep you updated about it.

Thanks
Lucas Costa

**Comment by larsga** [comment]
Great! 👍
